interface/pages/posts/new_person.tsx

```diff
@@ -30,92 +36,76 @@ export default function NewPerson() {
     validate: {
       name: (value) => (value ? null : 'Invalid name'),
       email: (value) => (/^\S+@\S+$/.test(value) ? null : 'Invalid email'),
-
     },
-
   });
 
   const postPerson = async (data: Omit<API.Person, "id">) => {
-    const response = await fetch(
-      `${URL}/persons/`,
-      {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(data),
-      }
-    )
-    const person = await response.json()
+    const response = await state.postPerson(data)
```

Comment:
Use the `state.post...` method instead of a direct `fetch` as it refreshes the data. (the new person will appear in the person table for ex)